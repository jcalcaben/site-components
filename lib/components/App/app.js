import React from 'react'

import AppContextProvider from '../../context/app';

const App = props => {

    const { children } = props;

    return <AppContextProvider>{children}</AppContextProvider>

}

export default App;