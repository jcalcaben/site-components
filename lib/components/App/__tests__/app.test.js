import React, { useEffect } from 'react';
import TestRenderer, { act } from 'react-test-renderer'

import App from '../app'

test('render app', ()=>{
    const Component = App;

    const testRenderer = TestRenderer.create(<Component><span>Content</span></Component>)

    expect(testRenderer.toJSON()).toMatchSnapshot();
})