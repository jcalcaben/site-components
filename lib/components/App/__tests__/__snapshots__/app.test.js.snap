// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render app 1`] = `
<span>
  Content
</span>
`;
